google/google_cloud_logging/spinnaker.conf

```diff
@@ -76,7 +75,7 @@
   format multiline
   # Match the date at the beginning of each entry
   format_firstline /^(\d+-\d+-\d+\s)/
-  format1 /(?<time>\d+-\d+-\d+ \d+:\d+:\d+\.\d+) +(?<severity>[^ ]+) +(?<pid>\d+) +--- + \[ *(<thread>.*) *\] (?<logger>[^ :]+) *:\s*(?<message>.*)/
```

Comment:
At least I think that's what's happening. This matches whereas the other
didnt. Is that inconsistent with "\[ *(<thread>.*) *\]" ?

On Thu, Mar 10, 2016 at 7:58 PM, Eric Wiseblatt <ewiseblatt@google.com>
wrote:

> It wasnt matching the regular expression because the " +" eagarly matches
> everything so the next " " doesnt match. I'd changed the expression shortly
> before submitting. When I tested it, I'd looked to see that entries in the
> console updated. But I wasnt paying attention that they were coming from
> other instances running with my older format (where I thought the '---' was
> a value field rather than just a separator).
>
> On Thu, Mar 10, 2016 at 7:52 PM, Lars Wander <notifications@github.com>
> wrote:
>
>> In google/google_cloud_logging/spinnaker.conf
>> <https://github.com/spinnaker/spinnaker/pull/789#discussion_r55775284>:
>>
>> > @@ -76,7 +75,7 @@
>> >    format multiline
>> >    # Match the date at the beginning of each entry
>> >    format_firstline /^(\d+-\d+-\d+\s)/
>> > -  format1 /(?<time>\d+-\d+-\d+ \d+:\d+:\d+\.\d+) +(?<severity>[^ ]+) +(?<pid>\d+) +--- + \[ *(<thread>.*) *\] (?<logger>[^ :]+) *:\s*(?<message>.*)/
>>
>> What does removing that space accomplish?
>>
>> —
>> Reply to this email directly or view it on GitHub
>> <https://github.com/spinnaker/spinnaker/pull/789/files#r55775284>.
>>
>
>